scrapers/vip4k.yml

```
@@ -0,0 +1,49 @@
+name: vip4k
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - black4k.com
+      - daddy4k.com
+      - fist4k.com
+      - hunt4k.com
+      - mature4k.com
+      - old4k.com
+      - rim4k.com
+      - sis.porn
+      - stuck4k.com
+      - tutor4k.com
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+     Title: //div[@class="title_player" or @class="title"]| //h1[contains(@class,"player")][contains(@class,"title")]
+     Performers:
+      Name:
+        selector: //a[@class="link_player"]/text()
+     Details:
+        selector: //div[@class="player-item__text"]/text() | //span[@class="player-info__text-area"]/text() |
+                  //span[@class="episode-about__text text"]/text() | //div[@class='desc_frame']/p/text() | //div[@class='wrap_player_desc']/p/text() |
+                  //div[@class='wrap_post']/p/text() | //div[@class="player-item__text text text--sm"]/text()
```

we can at least remove the /text() to clean this a bit
```suggestion
      Details:
        selector:
          //div[@class="player-item__text"]|//span[@class="player-info__text-area"]|//span[@class="episode-about__text text"]|//div[@class='desc_frame']/p|
          //div[@class='wrap_player_desc']/p|//div[@class='wrap_post']/p|//div[@class="player-item__text text text--sm"]
```